sections/nodes.include

```diff
@@ -98,6 +98,19 @@
 
 <p>A <a href="#concept-node">node</a> is considered <dfn>empty</dfn> if its <a href="#node-length">length</a> is zero.
 
+<h4>Document tree</h4>
+A <dfn concept>document tree</dfn> is a <a>node tree</a> whose <a>root</a> is a <a href="#concept-document">document</a>.
```

Comment:
Got an error link here

> LINK ERROR:[0m Multiple possible 'dfn' local refs for 'root'.
> Arbitrarily chose the one with type 'dfn' and for 'tree'.
> <a data-link-type="dfn" data-lt="root">root</a>

These are the definition of root in this spec:

> traversal.include:<p>The <dfn attribute id="iterator-root" for=NodeIterator>root</dfn> 
> terminology.include:<p>The <dfn id="tree-root" for="tree">root</dfn> 
> ranges.include:<p>The <dfn id="range-root" for="range">root of a range</dfn> 

Here shall be `<a href="#tree-root">root</a>`